pkg/reconciliation/construct_podtemplatespec.go

```diff
@@ -358,11 +360,14 @@ func getConfigDataEnVars(dc *api.CassandraDatacenter) ([]corev1.EnvVar, error) {
 	}
 
 	configData, err := dc.GetConfigAsJSON(dc.Spec.Config)
-
 	if err != nil {
 		return envVars, err
 	}
-	envVars = append(envVars, corev1.EnvVar{Name: "CONFIG_FILE_DATA", Value: configData})
+	cdcAdded, err := cdc.UpdateConfig(json.RawMessage(configData), *dc)
```

Comment:
As said above, either here check that if CDC isn't used - don't call it, or there exit quickly.

Reply:
We now exit quickly. I think this is addressed.